### DISCUSSION TITLE
Glossaries/Symbols not showing up. Long message instead.

### DISCUSSION THREAD

**Discussion by goncalopalacino** [joaomlourenco/novathesis#415] [Q&A] [answered]
Which version of the template are you using?
last version

Do you use a cloud-based service? Which one?
No. I use the Texshop(Mactex) program for macs.

What went wrong?
Error in symbols and chemical symbols, it only says the following:
![Captura de ecrã 2023-02-21, às 16 26 10](https://user-images.githubusercontent.com/125577805/220414712-d48356b3-6669-4384-8bee-49291b3d7d2d.png)
Both "template.slo" and "template.cho" files have the glossary entries, but no "template.sls" or "template.chs" files.
When searching for glossar in "template.log" this is what comes up:

![Captura de ecrã 2023-02-21, às 17 15 14](https://user-images.githubusercontent.com/125577805/220414810-8e2480f3-58a8-43ef-b1cb-db21363125b5.png)
What can i do to solve it ?

**Comment by joaomlourenco** [comment, marked as answer]
TeX Shop is not running "makeglossaries".
You have to go to preferences, select the "typesetting tab" and then "Default command" -> "ccommand listed below" -> "latexmk -pdf -shell-escape -synctex=1".
See screenshot below.
![image](https://user-images.githubusercontent.com/2064643/220750607-f790985e-64c8-4742-94c7-cbd817d9cff2.png)

**Comment by francisconeto00** [comment]
HI!
I'm having the same problem but I'm using Cocalc with the template latest version

**Reply by joaomlourenco** [reply]
That is a limitation of "cocalc"… it does not run `makeglossaries`, necessary for the glossary, list of symbols, etc.

**Comment by francisconeto00** [comment]


**Reply by joaomlourenco** [reply]
Please [follow the instructions](https://github.com/joaomlourenco/novathesis#getting-started) and then ask for more help if needed!  Thank you.

**Comment by joaomlourenco** [comment]
Dear @francisconeto00, You owe the developers the effort to make it work properly by reading **all the documentation** before asking for help.
**PLEASE** follow the instructions: https://github.com/joaomlourenco/novathesis/wiki/Uminho-Configuration
